setup.py

```diff
@@ -106,8 +106,9 @@
     'License :: OSI Approved :: Apache Software License',
 ]
 
-BUILD_WITH_BORING_SSL_ASM = os.environ.get('GRPC_BUILD_WITH_BORING_SSL_ASM',
-                                           True)
+# Environment variable to turn off BoringSSL ASM optimization.
+BUILD_WITHOUT_BORING_SSL_ASM = os.environ.get(
+    'GRPC_PYTHON_BUILD_WITHOUT_BORING_SSL_ASM', False)
```

Comment:
"Without" can be confusing because it results in double negatives.

Reply:
Let's postpone this PR. Yes, it is a bit confusing.